pkg/skaffold/initializer/init.go

```diff
@@ -44,7 +44,7 @@ func DoInit(ctx context.Context, out io.Writer, c config.Config) error {
 		return err
 	}
 
-	deployInitializer := deploy.NewInitializer(a.Manifests(), a.KustomizePaths(), c)
+	deployInitializer := deploy.NewInitializer(a.Manifests(), a.KustomizeBases(), a.KustomizePaths(), c)
```

Comment:
Maybe NewInitializer should just take the `ProjectAnalysis` instance?

Reply:
it causes an import cycle unfortunately. i'm sure there's a better way to do this but we can refactor later